signs-work-order-map/components/Map.tsx

```diff
@@ -14,22 +14,36 @@ interface LatLon {
   longitude: number;
 }
 
-export default function Map() {
+interface MapProps {
+  location: [number, number] | undefined;
+}
+
+export default function Map({ location }: MapProps) {
   const onDrag = useCallback((event: ViewStateChangeEvent) => {
     // truncate values to our preferred precision
-    const latitude = +event.viewState.latitude.toFixed(MAP_COORDINATE_PRECISION);
-    const longitude = +event.viewState.longitude.toFixed(MAP_COORDINATE_PRECISION);
+    const latitude = +event.viewState.latitude.toFixed(
+      MAP_COORDINATE_PRECISION
+    );
+    const longitude = +event.viewState.longitude.toFixed(
+      MAP_COORDINATE_PRECISION
+    );
     console.log(latitude, longitude);
     setMapLatLon({
       latitude,
       longitude,
     });
+    // send location to Knack
+    window.parent.postMessage(
```

Comment:
part of me thinks that wrapping this postMessage function as a function that is returned from the `useIFrameMessenger` custom hook might help us complete the loop on sending and receiving messages but idk. 

I also go wild with grouping things in custom hooks sometimes...